community/cloud-foundation/templates/iam_member/tests/integration/iam_member.yaml

```diff
@@ -1,7 +1,7 @@
-# Example on how to use the iam member template
+# Test of the IAM member template .
 #
 # Variables:
-#   RAND: A random string used mostly by the testing suite.
+#   RAND: A random string used by the testing suite
```

Comment:
we should also test using 'user', 'domain' and, 'group'. 
Can receive those from the configuration file?

Reply:
@fawix Users/Domains/Groups are difficult to test using the current BATS automated testing as each entity needs to be created within the test each time. These entities are managed by GSuite.

Reply:
Can't it be a prerequisit for the test, like the existance of the org? We are not testing the creation of  Users/Domains/Groups, only the ability to attache roles to them if they exist. I know it creates external dependencies, but it's better then not testing it at all.

Reply:
I definitely agree that these should be tested. This can be applied to any of the current templates that modify IAM permissions. Perhaps this should be addressed in a future commit as this PR is for documentation edits.